### ISSUE TITLE
Running tests with multiple "--coverageReporters" options

### ISSUE THREAD

**Issue by jderrough** [redwoodjs/redwood#3419]
Running `yarn rw test --coverage --coverageReporters=html --coverageReporters=text` results in:

```
Failed to write coverage reports:
ERROR: Error: Cannot find module 'html,text'
````

Looking at the command line, what is actually executed is something like:
```
jest --coverage true --coverageReporters html,text --passWithNoTests
```

It looks like the multiple "--coverageReporters" option values are joined into one with a "," separator, which is not supported by jest.

**Comment by InbarRose** [comment]
any update on this?

**Comment by thedavidprice** [comment]
This will be resolved in #4096
